locales/nl_NL.json

```diff
@@ -398,105 +398,105 @@
+          "message-no-state": "We laten je per e-mail weten wanneer je status volledig wordt ondersteund. Neem bij vragen <0> contact op met ons ondersteuningsteam </0>."
         },
         "generic": {
-          "title": "Not all Nash services are currently supported in your present location. Your access to some parts of the Nash platform will be restricted.",
-          "message": "Please <0>contact our support team</0> with any questions."
+          "title": "Niet alle Nash-services worden momenteel ondersteund op uw huidige locatie. Uw toegang tot sommige delen van het Nash-platform is beperkt.",
+          "message": "Neem bij vragen <0> contact op met ons ondersteuningsteam </0>."
         }
       },
       "tier-too-low": {
-        "title": "Your current verification level is Tier {{userTier}}. As a result, your access to some parts of the Nash platform will be restricted.",
-        "message": "To use our full portfolio of services, please <0>upgrade to Tier {{minTier}}</0>."
+        "title": "Uw huidige verificatieniveau is niveau {{userTier}}. Als gevolg hiervan is uw toegang tot sommige delen van het Nash-platform beperkt. ",
+         "message": "Om ons volledige dienstenportfolio te gebruiken, moet u <0> upgraden naar Tier {{minTier}} </0>."
       }
     },
     "info": {
       "tier-1+": {
         "location-mismatch": {
-          "title": "We noticed you’re signing in from {{detectedCountry}}.",
-          "message": "If you no longer live in {{accountCountry}}, please <0>contact our support team</0> to update your address."
+          "title": "We merken dat u zich aanmeldt vanuit {{detectedCountry}}.",
+          "message": "Als u niet langer in {{accountCountry}} woont, neem dan <0> contact op met ons ondersteuningsteam </0> om uw woonplaats bij te werken."
         }
       },
       "browser": {
         "use-desktop-for-advanced-exchange": {
-          "title": "Browse on desktop for a better experience.",
-          "message": "The ‘Pro’ view is designed for desktop usage. If you are on mobile, try the <0>‘Basic’ view</0>."
+          "title": "Gebruik een desktop voor een betere ervaring.",
+          "message": "De ‘Pro’ -weergave is ontworpen voor desktopgebruik. Als u op uw mobiel bent, probeer dan de <0> ‘Basis’ weergave </0>."
         }
       }
     },
-    "dismiss-message": "Don’t tell me this again."
+    "dismiss-message": "Vertel mij dit niet nog een keer."
   },
   "terms": {
-    "user-agreement": "User Agreement",
-    "privacy-notice": "Privacy Notice"
+    "user-agreement": "Gebruikersovereenkomst",
+    "privacy-notice": "Privacyverklaring"
   },
   "account-create": {
     "steps": [
-      "Your details",
-      "Country",
-      "Account details"
+      "Jouw details",
+      "Land",
+      "Accountgegevens"
     ],
     "signin": {
-      "desc": "Already have a Nash account?",
-      "sign-in-now": "Sign in now"
+      "desc": "Al een Nash account?",
```

Comment:
Eventueel: 'Heb je al een Nash account?' of 'Nog geen Nash account?'

Reply:
'Heb je al een Nash account?' staat inderdaad beter.